pydantic/json_schema.py

```diff
@@ -1609,13 +1609,13 @@ def json_schema(self, schema: core_schema.JsonSchema) -> JsonSchemaValue:
         Returns:
             The generated JSON schema.
         """
+        content_core_schema = schema['schema'] if 'schema' in schema else core_schema.any_schema()
```

Comment:
```suggestion
    content_core_schema = schema.get('schema') or core_schema.any_schema()
```
I think this is the "better" version of your suggestion, which works since all schemas must have a type key so `schema.get('schema')` will be falsy only if it is `None` (and having the value None is not meaningfully different from not being present)